ucs/adapters/outbound/rabbitmq_produce.py

```diff
@@ -20,30 +20,25 @@
 from ghga_message_schemas import schemas
 from ghga_service_chassis_lib.pubsub import AmqpTopic, PubSubConfigBase
 
-from ucs.domain import models
-from ucs.domain.interfaces.outbound.event_pub import IEventPublisher
+from ucs.core import models
+from ucs.ports.outbound.event_pub import IEventPublisher
+
+
+class RMQPublisherConfig(PubSubConfigBase):
+    """Config parameters and their defaults."""
+
+    topic_upload_received: str = "file_upload_received"
 
 
 class RabbitMQEventPublisher(IEventPublisher):
```

Comment:
```suggestion
from ucs.ports.outbound.event_pub import EventPublisher


class RMQPublisherConfig(PubSubConfigBase):
    """Config parameters and their defaults."""

    topic_upload_received: str = "file_upload_received"


class RabbitMQEventPublisher(EventPublisher):
```
I was removed in other places, should be consistent

Reply:
This is code that will be removed in the next step, thus polishing it might not be the most important step